statsforecast/core.py

```diff
@@ -1686,6 +1693,33 @@ def plotly(
                 if n_cols == 1:
                     axes = np.array([axes])
 
+            def plotly(
```

Comment:
Why is this function being defined here?

Reply:
I really don't know. The original code that had defined there, I tried not to move it.

Reply:
It's the equivalent here: https://github.com/Nixtla/statsforecast/blob/main/statsforecast/core.py#L1534

Reply:
Ah, I understand! Great catch. Will update.

Reply:
I have updated it